vacation/vacation.go

```diff
@@ -16,9 +17,17 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/gabriel-vasile/mimetype"
 	"github.com/ulikunitz/xz"
 )
 
+// An Archive decompresses tar, gzip and xz compressed tar, and zip files from
+// an input stream.
+type Archive struct {
+	reader     io.Reader
+	components int
+}
+
 // A TarArchive decompresses tar files from an input stream.
 type TarArchive struct {
 	reader     io.Reader
```

Comment:
Do we still need TarArchive / NewTarArchive if Archive covers decompressing tar files?